src/lib/api/group/client/groupApiClient.client.test.ts

```diff
@@ -146,24 +146,35 @@ describe('API Function Tests', () => {
   describe('putGenerateInvitationLink', () => {
     //mock params and response
     const mockGroupId = '6ec791b9-945c-4c1c-a872-3610521650e4';
-    const mockInvitationLinkHash = 'e8ee3bc535b0c569276a801de8a3fd88';
-    const mockGenerateInvitationLinkResponse = JSON.stringify({
-      invitationLinkHash: mockInvitationLinkHash,
-    });
 
     it('successfully generates invitation link', async () => {
-      const mockRequest = setUpMockSuccessRequest(mockGenerateInvitationLinkResponse);
+      /* Arrange */
+      const mockInvitationLinkHash = 'e8ee3bc535b0c569276a801de8a3fd88';
+      const mockResponse = JSON.stringify({
+        invitationLinkHash: 'e8ee3bc535b0c569276a801de8a3fd88',
```

Comment:
```suggestion
        invitationLinkHash: mockInvitationLinkHash,
```
This is better as it used to be